vermouth/tests/test_molecule.py

```diff
@@ -931,10 +990,23 @@ def random_interaction(draw, graph, natoms=None,
 @st.composite
 def interaction_collection(draw, graph,
                            interaction_class=Interaction, attrs=False):
+    """
+    Strategy that builds a collection of interaction-like intances.
+
+    The collection is a dictionary with any string as key, and a list of
+    :class:`~vermouth.molecule.Interaction` or
+    :class:`~vermouth.molecule.Interaction`.
+
+    The parameters are passed to :func:`random_interaction`.
```

Comment:
Shouldn't they be *args and **kwargs then?

Reply:
`*args` and `**kwargs` should be avoided when possible, they deactivate many health checks on the arguments. Because we know exactly what the arguments re here, we can ask for them explicitly.